chargeio/services/Transactions/TransactionService.cs

```diff
@@ -1,27 +1,18 @@
 ﻿using System;
 using System.Collections.Generic;
+using ChargeIo.services;
 using ChargeIO.Infrastructure;
 using ChargeIO.Models;
 
 namespace ChargeIO.Services.Transactions
 {
-	public class TransactionService
-	{
-        private string SecretKey { get; }
-
-        public TransactionService(string secretKey = "")
-		{
-		    if (secretKey == null)
-		    {
-		        SecretKey = Configuration.SecretKey;
-		    }
-		    else
-		    {
-		        SecretKey = secretKey.Length > 0 ? secretKey : Configuration.SecretKey;
-		    }
-		}
-
-        public virtual Charge Charge(ChargeOptions options)
+    public sealed class TransactionService : ServiceBase

```

Comment:
We shouldn't be sealing our classes -- we shouldn't be restricting users from inheriting from our classes.